src/languages/ebnf.js

```diff
@@ -9,7 +9,7 @@ export default function(hljs) {
 
     var nonTerminalMode = {
         className: "attribute",
-        begin: /^[ ]*[a-zA-Z][a-zA-Z-_]*([\s-_]+[a-zA-Z][a-zA-Z]*)*/
+        begin: /^[ ]*[a-zA-Z][a-zA-Z-_]*([\s\-_]+[a-zA-Z][a-zA-Z]*)*/
```

Comment:
```js
begin: /^[ ]*[a-zA-Z][a-zA-Z-_]*([\s\-_]+[a-zA-Z][a-zA-Z]*)*/
#                           ^       ^^
```

You're escaping one `-` here but not the other?

Reply:
Doesn't need to be, but I'll move them both to the end to make this consistent.